guides/source/association_basics.md

```diff
@@ -783,7 +811,10 @@ The following sections give the details of each type of association, including t
 
 ### `belongs_to` Association Reference
 
-The `belongs_to` association creates a one-to-one match with another model. In database terms, this association says that this class contains the foreign key. If the other class contains the foreign key, then you should use `has_one` instead.
+In database terms, the `belongs_to` association says that this model's table contains a column which represents a reference to another table.
+This can be used to set up one-to-one or one-to-many relations, depending on the setup.
+If the table of the other class contains the reference in a one-to-one relation, then you should use `has_one` instead.
+
```

Comment:
At a first glance, this sounds precise, and that's a worthy goal. However, it seems as though we could be making it harder to comprehend for people that are new to the material. Perhaps we could reinforce it with a diagram instead?

Reply:
Here it's the second more detailed description of `belongs_to`, at the very beginning there's a simplified one with a diagram already (line 89). I doubt we should have almost exactly the same images repeated. Reference part is more likely to be read by people who are returning to guide for details and this should not be too hard for them at this moment.
For first-timers we cannot expect 100% comprehension anyway, but can plant an idea to re-read after some time.

Reply:
I've looked at this from the beginning of the file and had a look at the rendered Markdown and it reads well.